packages/theme/pages/Checkout/Customer.vue

```diff
@@ -140,6 +142,18 @@ export default {
       isFormSubmitted.value = true;
     };
 
+    onMounted(async () => {
```

Comment:
Maybe  its better to load that onSSR, this will verify quicker if user has access to this page

Reply:
I tried to put it in onSSRc lifecycle hook at first but it did not work. Now when you have already filled the customer step data and you are in other step (like shipping) when you come back to customer the data will be filled for you.